### PR TITLE
Fix a typo in docs/configuration/configuration.md

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@fujitsu.com>

### CODE REVIEW

File: docs/configuration/configuration.md [hunk 1]
```diff
@@ -1603,7 +1603,7 @@ Available meta labels:
   from underlying pods), the following labels are attached:
 * `__meta_kubernetes_endpointslice_address_target_kind`: Kind of the referenced object.
 * `__meta_kubernetes_endpointslice_address_target_name`: Name of referenced object.
-* `__meta_kubernetes_endpointslice_address_type`: The ip protocol family of the adress target.
+* `__meta_kubernetes_endpointslice_address_type`: The ip protocol family of the address target.
```
Comment (LeviHarrison) [inline docs/configuration/configuration.md:1606]: I believe this description intends to indicate the address possessed by the target, so we can make it more clear by fixing the grammar.

```suggestion
* `__meta_kubernetes_endpointslice_address_type`: The ip protocol family of the address of the target.
```

**Review by LeviHarrison — COMMENTED** [review]
Thanks! Just one more improvement we could make to this line.

File: docs/configuration/configuration.md (hunk 1 above)
Comment (hs0210) [reply docs/configuration/configuration.md]: Thanks! Done. PTAL.

**Review by LeviHarrison — APPROVED** [review]
LGTM

**Comment by LeviHarrison** [comment]
Thanks!

**Pull request merged**